deepomatic/cli/output_data.py

```
@@ -54,31 +54,32 @@ def get_output(descriptor, kwargs):
         return DisplayOutputData(**kwargs)
 
 
-def get_outputs(descriptors, kwargs):
-    return [get_output(descriptor, kwargs) for descriptor in descriptors]
+def get_outputs(kwargs):
+    descriptors = kwargs.get('outputs', [])
+    # Update output fps to default value if none was specified.
```

Maybe raise an error?